frontend/app/controllers/Event.scala

```diff
@@ -160,11 +160,11 @@ class Event(
   }
 
   def eventbriteRedirect(event: RichEvent, discountCodeOpt: Option[EBCode])(implicit req: RequestHeader) = {
-    val eventUrl = discountCodeOpt.fold(Uri.parse(event.url))(c => event.url ? ("discount" -> c.code))
+    val eventUrl = discountCodeOpt.fold(Uri.parse(event.underlying.ebEvent.url))(c => event.underlying.ebEvent.url ? ("discount" -> c.code))
     Found(addEventBriteGACrossDomainParam(eventUrl)).withCookies(Cookie(eventCookie(event), "", Some(3600)))
   }
 
-  @deprecated("Guardian Live Events are not configured to use this anymore, and host their own thankyou page on EventBrite.")
+  @deprecated("Guardian Live Events are not configured to use this anymore, and host their own thankyou page on EventBrite.", "")
```

Comment:
the single argument deprecated annotation is deprecated - the second argument is to say which version it's deprecated since